app/views/govuk_publishing_components/components/docs/chart.yml

```diff
@@ -137,6 +137,77 @@ examples:
             - 118
             - 85
             - 80
+  with_a_different_point_size:
+    description: For some charts it may be better to reduce the point size so that the line is clearer. However it is recommended that the point size is larger to make it easier for users to move the mouse over them.
+    data:
+      chart_heading: Page views chart
+      h_axis_title: Day
+      v_axis_title: Views
+      chart_overview: This chart shows page views for January 2015.
+      point_size: 0
+      keys:
+        - "2015-01-01"
+        - "2015-01-02"
+        - "2015-01-03"
+        - "2015-01-04"
+        - "2015-01-05"
+        - "2015-01-06"
+        - "2015-01-07"
+        - "2015-01-08"
+        - "2015-01-09"
+        - "2015-01-10"
+      rows:
+        - label: January 2015
+          values:
+            - 600
+            - 690
+            - 740
+            - 820
+            - 570
+            - 950
+            - 810
+            - 610
+            - 770
+            - 530
+  scale_y_axis_based_on_data:
+    description: By default the Y-axis will always start at zero, but this may not always be required. Note also that if passed gaps in the data, the chart will simply not render points and discontinue the line.
```

Comment:
Nitpick: I found `Note also that if passed gaps in the data` a bit confusing to read the first couple times, maybe it could be reworded slightly?

Reply:
That was a bit of a mess, wasn't it? I've reworded 👍

Reply:
Thanks @andysellick 👍